apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -2332,6 +2451,7 @@ static void libc_stdio_ungetc_tc(void)
 	char *filename = VFS_FILE_PATH;
 	int ret;
 	int ch1, ch2;
+	int origin_fs_nungotten;
```

Comment:
int num will be enough for number of characters.

Reply:
Also it used only below condition, please add here.
if CONFIG_NUNGET_CHARS > 0

Reply:
done